maestro-test-scripts/src/main/groovy/multipoint/FixedRateTest.groovy

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2018 Otavio R. Piske <angusyoung@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package singlepoint
+
+@GrabConfig(systemClassLoader=true)
+
+@Grab(group='commons-cli', module='commons-cli', version='1.3.1')
+@Grab(group='org.apache.commons', module='commons-lang3', version='3.6')
+
+@Grab(group='org.msgpack', module='msgpack-core', version='0.8.3')
```

Comment:
The test scripts don't need these dependencies anymore: commons-cli, commons-lang3 and msgpack-core. You can remove those.